### PR TITLE
Save `soss_simulator` input data to output FITS files

### PR DESCRIPTION
**Pull request by hover2pi**
This improvement stores the input `star` and `planet` data (including the orbital parameters) to the output FITS files so users can make direct comparisons. The data is stored in a new 'STAR' and (when applicable) 'PLANET' extensions with the units and orbital params saved as header keywords.

This also adds the `soss_simulator.SossSim.plot_input` method to allow easy visual inspection of the input data.

### CODE REVIEW

**Comment by hover2pi** [comment]
Hi @bhilbert4 ! Looks like these tests are failing because they don't contain the IRSA fix you added in #772 . Will you be merging that soon so I can pull it into here from `master`? Thanks!

**Comment by bhilbert4** [comment]
@hover2pi sorry about that. Commissioning shifts are stealing all my attention these days, at the expense of Mirage and other things. Could you give #772 a quick review? I think it is ready to go.  I'll give this one a look over. Also, if you have time, do you think you could review #767? That one is basically an updated rotation equation.

**Comment by bhilbert4** [comment]
#772 has been merged, so you should be able to bring those changes into this branch now.

File: mirage/catalogs/model_atmosphere.py [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+
+    # Resolution of input spectra in the same wavelength range:
+    idx = np.where((input_w >= min_wav - 10) & (input_w <= max_wav + 10))[0]
+    res = get_resolution(input_w[idx], input_f[idx])
+
+    # If input spetrum resolution is larger, degrade:
+    if res > resT:
+
+        # This can be way quicker if we just take the gaussian weight *at* the evaluated
+        # points in the interpolation. TODO: make faster.
```
Comment (bhilbert4) [inline mirage/catalogs/model_atmosphere.py:458]: Did you mean to do this now, or in a future PR?

**Review by bhilbert4 — APPROVED** [review]
Looks good to me, as soon as the tests are passing.

**Pull request merged**